.github/workflows/r-python-interop-testing.yml

```diff
@@ -27,19 +27,16 @@ jobs:
       - name: Checkout TileDB-SOMA
         uses: actions/checkout@v4
         with:
-          fetch-depth: 0  # ensure we get all tags to inform package version determination
+          fetch-depth: 0 # ensure we get all tags to inform package version determination
 
       - name: Bootstrap
         run: cd apis/r && tools/r-ci.sh bootstrap
 
       - name: Dependencies
         run: cd apis/r && Rscript -e "remotes::install_deps(dependencies = TRUE, upgrade = FALSE)"
 
-      - name: CMake
```

Comment:
Unclear to me why mods in this file are on this PR

Reply:
Because without these mods, the PR won't build.

Reply:
Sorry I clicked on the wront lines. I'll re-comment.